packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -125,6 +125,8 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
 
   private _isScrollIdle: boolean;
 
+  private _isInPreview: boolean;
```

Comment:
What is `_isInPreview`? Is it true when there's a pending value?

Reply:
This is when the pending values is not undefined